blockchain/src/main/java/com/tangem/blockchain/blockchains/optimism/OptimismWalletManager.kt

```diff
@@ -66,7 +64,11 @@ class OptimismWalletManager(
         lastLayer1FeeAmount = lastLayer1Fee
 
         //https://community.optimism.io/docs/developers/build/transaction-fees/#displaying-fees-to-users
-        val updatedFees = layer2fee.map { it.copy(value = it.value!! + lastLayer1Fee.value!!) }
+        val updatedFees = layer2fee.copy(
+            minimum = layer2fee.minimum + requireNotNull(lastLayer1Fee.value),
```

Comment:
Думаю стоит сообщение добавить, а то получается, что нет различия с `!!`

```suggestion
            minimum = layer2fee.minimum + requireNotNull(lastLayer1Fee.value) { "Last layer 1 fee must not be null" },
```

Reply:
готово в https://github.com/tangem/blockchain-sdk-kotlin/pull/218